combat-trainer.lic

```diff
@@ -2133,6 +2218,12 @@ class GameState
     DRStats.fatigue < @fatigue_regen_threshold
   end
 
+  def balance_low?
+    echo("***Balance: #{DRStats.balance}***") if $debug_mode_ct
+    echo("***Target: #{@balance_regen_threshold}***") if $debug_mode_ct
+    DRStats.balance < @balance_regen_threshold
```

Comment:
Is the balance threshold change necessary for whirlwind, or could it be a separate PR?